Client/Frontend/Strings.swift

```diff
@@ -82,6 +82,30 @@ extension String {
                 value: "Restore tabs",
                 comment: "The title for the affirmative action of the restore tabs pop-up alert. This alert shows when opening up Firefox after it crashed, and will restore existing tabs.")
         }
+
+        public struct FeltDeletion {
+            public static let Title = MZLocalizedString(
+                key: "Alerts.FeltDeletion.Title.v122",
+                tableName: "Alerts",
+                value: "End your private session?",
+                comment: "When tapping the fire icon in private mode, an alert comes up asking to confirm if you want to delete all browsing data and end your private session. This is the title for the alert.")
+            public static let Body = MZLocalizedString(
+                key: "Alerts.FeltDeletion.Body.v122",
+                tableName: "Alerts",
+                value: "Close all private tabs and delete history, cookies, and all other site data.",
```

Comment:
did we get clarify whether we are deleting only this session or all data? 

we discussed how this part of deleting `all` data contradicts with deleting only session data, do we think we'll need to make a string change in this case?

Reply:
We're good to leave it. Just got the word on that.